components/ScreenSizePicker/index.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { css } from 'glamor'
+import {
+  plainButtonRule,
+  fontStyles,
+  useColorContext
+} from '@project-r/styleguide'
+
+import MdPhoneIphone from 'react-icons/lib/md/phone-iphone'
+import MdTabletMac from 'react-icons/lib/md/tablet-mac'
+import MdLaptopMac from 'react-icons/lib/md/laptop-mac'
+import MdDesktopMac from 'react-icons/lib/md/desktop-mac'
+import MdFullscreen from 'react-icons/lib/md/fullscreen'
+
+const styles = {
+  container: css({
+    display: 'flex',
+    justifyContent: 'flex-start',
+    flexWrap: 'wrap'
+  }),
+  iconContainer: css({
+    flex: '0 0 30%',
+    minWidth: 80,
+    padding: '16px 0',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center'
+  }),
+  label: css({
+    ...fontStyles.sansSerifRegular12,
+    marginTop: 4
+  })
+}
+
+const screenSizes = [
+  {
+    name: 'phone',
+    label: 'Mobile',
+    Icon: MdPhoneIphone
+  },
+  {
+    name: 'tablet',
+    label: 'Tablet',
+    Icon: MdTabletMac
+  },
+  {
+    name: 'laptop',
+    label: 'Laptop',
+    Icon: MdLaptopMac
+  },
+  {
+    name: 'desktop',
+    label: 'Desktop',
+    Icon: MdDesktopMac
+  },
+  {
+    name: null,
+    label: 'Editor',
+    Icon: MdFullscreen
```

Comment:
I am wondering about the use of the editor view here and how it integrates with the rest of the options. The writer can write, but not insert new elements (which are kind of the problematic ones as far as responsiveness goes). Does it make sense to add a second, incomplete version of the editor? Should that maybe reset the sidebar to `editieren`? But do we need it then? Happy to hear your thoughts.

Reply:
Yeah I think it should not be there.